services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -297,25 +303,43 @@ export default class IndexPage extends React.Component<Props> {
                       className="txt-f txt-f--sm"
                       size={10}
                       placeholder="Confirm Email"
+                      onBlur={this.handleBlur}
                     />
                   </div>
                 </div>
                 <div className="bc bc--r p-t500">
-                  <button type="submit" className="btn btn--700">
+                  <button
+                    type="submit"
+                    className="btn btn--700"
+                    onSubmit={this.handleSubmit}
+                  >
                     Send Message
                   </button>
+                  {this.emailError()}
                 </div>
               </div>
             </fieldset>
           </form>
-
           <ul>
-            {commissions.map(commission => (
-              <li key={commission.id}>{commission.name}</li>
-            ))}
+            {commissions.map(commission => this.renderCommission(commission))}
           </ul>
         </div>
       </div>
     );
   }
+  emailError() {
```

Comment:
Rename to `renderEmailError` since it returns HTML.

Also add newlines between methods.